src/uct/tcp/tcp_iface.c

```diff
@@ -221,7 +230,8 @@ static ucs_status_t uct_tcp_iface_query(uct_iface_h tl_iface,
                              sizeof(uct_tcp_am_hdr_t);
     ucs_status_t status;
     int is_default;
-    char sysfs_path[PATH_MAX];
+    char path_buffer[PATH_MAX];
```

Comment:
Can allocate it in `uct_tcp_iface_get_sysfs_path` instead?

Reply:
you mean with malloc?

Reply:
IMO it's worse to malloc because then malloc and free won't be in the same function